rep-2001.rst

```diff
@@ -773,6 +775,34 @@ Desktop Full
       packages: [perception, simulation, ros_ign_gazebo_demos]
 
 
+ROS Distribution Independent (September 2022 - Ongoing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These variants are not released as ROS packages within each distribution as the others are.
+Instead they are made available as independent packages outside of any ROS distribution.
+
+Build Essential
+"""""""""""""""
+
+In addition to the explicitly listed packages this variant should include the base toolchain for any operating system (such as `build-essential`) on Debian as well as GNU Make and a C++ compiler if not already part of that toolchain.
```

Comment:
```suggestion
In addition to the explicitly listed packages this variant should include the base toolchain for any operating system (such as `build-essential` on Debian) as well as GNU Make and a C++ compiler if not already part of that toolchain.
```

Besides my edits, I'm wondering if we should remove the `GNU Make` reference; someday in the future when we make a Windows version of that, it won't be accurate.  On the other hand, I'm OK with deferring this until we get to that point.

Reply:
> I'm wondering if we should remove the GNU Make reference; someday in the future when we make a Windows version of that, it won't be accurate. On the other hand, I'm OK with deferring this until we get to that point.

I think it would make more sense to make Windows the special case (since the MSVC toolchain would be the requirement instead). I've had issues building CMake projects in the default ROS configurations when BSD make is the default instead of GNU make.